### DISCUSSION TITLE
Can Neper2CAE only be used to generate 3D models?

### DISCUSSION THREAD

**Discussion by cherry12306** [neperfepx/neper#766] [Q&A]
Can Neper2CAE (https://github.com/ngrilli/Neper2CAE) only be used to generate 3D models? Can't it be used to generate 2D models? If it can be used to generate 2D model, how should I change this code: GeneratePolycrystal.py? I tried to modify it to 2D, but I'm having trouble with 'p.PartitionCellByExtrudeEdge(line=sweepEdge, cells=pickedCells, edges=tempedge, sense=REVERSE)'. How should I modify this partition in 2D?

**Comment by rquey** [comment]
Neper2CAE is not part of Neper itself. So, we cannot help.

**Reply by cherry12306** [reply]
Okay, Thanks for your reply, I will try again.
